client/components/common/FileUpload.vue

```diff
@@ -61,7 +62,7 @@ export default {
       this.$validator.validate(this.id).then(isValid => {
         if (!isValid) return;
         this.uploading = true;
-        return api.upload(this.form)
+        return this.$storageService.upload(this.form)
```

Comment:
I feel bad writing it here because, **again**, it is not your fault but this whole `this.createFileForm(e) => this.form` charade can be replaced with:
```js
const formData = new FormData(e.target);
```